R-package/cran-comments.md

```diff
@@ -674,7 +674,7 @@ Failing pre-checks.
 
     New submission
 
-    Possibly mis-spelled words in DESCRIPTION:
+    Possibly misspelled words in DESCRIPTION:
```

Comment:
These should be reverted. This is the exact text of what `R CMD check` produced. I think that for historical purposes, and for code search purposes, it's important that this remain an **exact** record of those log messages.